src/patternfly/patternfly.scss

```
@@ -4,3 +4,30 @@
 @use './patternfly-base';
 @use './components';
 @use './layouts';
+
+:where(.pf-v6-theme-glass) {
+  // custom stuff for the POC
+  .pf-v6-c-page, .pf-v6-c-login, body {
+    background-image: url("./assets/images/glass-brand-light.jpg");
+    // background-image: url("./assets/images/compass--wallpaper-light.jpg");
```

Stylelint (17.4.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)